### PR TITLE
Maintenance/changes from master

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- sslheader middleware is not happy when there are hyphens in its name
- for local deploys a port 10081 is opened until the AWS PR comes in

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

File: services/simcore/.env [hunk 1]
```diff
@@ -58,9 +58,11 @@ STORAGE_ENDPOINT=${PREFIX_STACK_NAME}_storage:8080
 STORAGE_HOST=${PREFIX_STACK_NAME}_storage
 
 SWARM_STACK_NAME=${PREFIX_STACK_NAME}-simcore
+SWARM_STACK_NAME_NO_HYPHEN=${PREFIX_STACK_NAME}_simcore
```
Comment (pcrespov) [inline services/simcore/.env:61]: what if ``PREFIX_STACK_NAME`` has hyphen? Cannot we use [shell parameter expansions](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) in the makefile for ``PREFIX_STACK_NAME``?

File: services/deployment-agent/.env [hunk 2]
```diff
@@ -7,4 +7,4 @@ SIMCORE_STACK_NAME=simcore
 # defines the dockerhub registry organisation
 SIMCORE_DOCKER_REGISTRY=itisfoundation
 # set the image tag to be used from dockerhub
-SIMCORE_IMAGE_TAG=master-latest
+SIMCORE_IMAGE_TAG=master-github-latest
```
Comment (pcrespov) [inline services/deployment-agent/.env:10]: why the ``gitbhu`` reference? As a default to this repo vs the environment repos in gitlab?

Comment (sanderegg) [reply services/deployment-agent/.env:10]: that are the images generated by github actions

File: services/simcore/.env (hunk 1 above)
Comment (sanderegg) [reply services/simcore/.env:61]: yes that is a good point. but normally these are named as master, staging or production

**Pull request merged**